src/state/action-creators.js

```diff
@@ -18,6 +18,7 @@ import type {
 import noImpact from './no-impact';
 import getNewHomeClientCenter from './get-new-home-client-center';
 import { add, subtract, isEqual } from './position';
+import { error } from '../logger/logger';
```

Comment:
For all the imports I think it will read better if we do:

```js
import * as log from '../logger/logger';

log.error('my error');
```

Otherwise it might look like we are calling an error handler